docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -536,6 +537,23 @@ Specifies your computer's hostname. It is used when running tests in [remote bro
 
 If the hostname is not specified, TestCafe uses the operating system hostname or network IP address of the current machine.
```

Comment:
If the hostname is not specified, TestCafe uses the operating system hostname or network IP address of the current machine. -> If the hostname is not specified, TestCafe uses the operating system's hostname or the current machine's network IP address.